ocaml/file_formats/cms_format.ml

```diff
@@ -120,7 +120,7 @@ let save_cms filename modname binary_annots sourcefile initial_env shape =
             cms_source_digest = source_digest;
             cms_initial_env = if Cmt_format.need_to_clear_env
               then Env.keep_only_summary initial_env else initial_env;
-            cms_uid_to_loc;
+            cms_uid_to_loc = cms_uid_to_loc |> Shape.Uid.Tbl.to_list |> Array.of_list;
```

Comment:
I'm a tiny bit worried about the performance cost of this indirect conversion. Can we go straight to an array?